ex1/ex1-index.js

```diff
@@ -0,0 +1,40 @@
+// **Product rezolvă exercițiile într-un mediu de programare la alegere (ex. Replit, VS Code)**  **și încarcă codul pe Replit.**
+// **1. Scrieți o funcție care returnează pătratul unui număr**
+// Funcția `findSquare(num)` returnează pătratul numărului primit ca date de intrare (este transmis ca argument în apelul funcției).
+// Exemplu 1:
+// `Input: num = 6`
+// `Output: 36`
+// Exemplu 2:
+// `Input: num = 0`
+// `Output: 0`
+// Exemplu 3:
+// `Input: num = -12`
+// `Output: 144`
+//..................................................................................................................................
+
+console.clear()
+
+function theSquareOfInput1(number) { 
+
+       if (typeof number === 'number' && !isNaN(number)) {
```

Comment:
```suggestion
    if (typeof number === 'number' && !isNaN(number)) {
```
Vezi ca ai cam multe spatii ca si identare.
De obicei 4 spatii sunt suficiente si e un bund standard